src/components/Tokens/TokenSelector/index.tsx

```diff
@@ -19,67 +19,57 @@ interface TokenOption {
 }
 
 interface Props {
+  variant: string;
   tokenOptions: Array<TokenOption>;
+  showBalances: boolean;
+  currentToken: TokenOption;
+  onChange: (type: TokenType) => void;
 }
 
 const TokenSelector = ({
-  tokenOptions
+  variant,
+  tokenOptions,
+  currentToken,
+  onChange,
+  showBalances
 }: Props): JSX.Element => {
-  const getTokenOption =
-    React.useCallback(
-      (type: TokenType) => tokenOptions.find(token => token.type === type),
-      [tokenOptions]
-    );
-
-  const [currentToken, setCurrentToken] = React.useState<TokenOption | undefined>(getTokenOption(TokenType.COLLATERAL));
-
-  const handleSelectToken = (selectedToken: TokenType) => {
-    setCurrentToken(getTokenOption(selectedToken));
-  };
-
-  // This is required to ensure that the displayed balance from the selected
-  // token options is kept updated, e.g. if a token is funded or if the initial
-  // balance is updated after the first render.
-  React.useEffect(() => {
-    if (!currentToken) {
-      return;
-    }
-
-    const selectedTokenType = currentToken.type;
-
-    setCurrentToken(getTokenOption(selectedTokenType));
-  }, [
-    currentToken,
-    getTokenOption
-  ]);
-
   return (
     <>
       {currentToken && (
         <Select
+          variant={variant}
           key={currentToken.type}
           value={currentToken.type}
-          onChange={handleSelectToken}>
+          onChange={onChange}>
           {({ open }) => (
             <>
               <SelectBody
                 className={clsx(
                   'w-52'
                 )}>
-                <SelectButton>
+                <SelectButton variant={variant}>
                   <span
                     className={clsx(
+                      variant === 'formField' ? 'py-2' : null,
                       'flex',
+                      variant === 'formField' ? 'text-xl' : null,
```

Comment:
@tomjeatt 
It could be a minor thing but related to a practice and look a little more correct.
It could be `{ 'formField': variant === 'formField' }`.